assets/src/dashboard/components/pageStructure/navigationComponents.js

```diff
@@ -43,72 +42,73 @@ export const Header = styled.h1`
   }
 `;
 
-export const NavButton = styled(Button)`
-  margin-bottom: 0;
-  margin-top: 0;
-  background-color: ${({ theme }) =>
-    theme.DEPRECATED_THEME.colors.foreground.gray12};
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.black};
-  border-radius: 4px;
-  border-color: ${({ theme }) =>
-    theme.DEPRECATED_THEME.colors.foreground.gray12};
-  height: 36px;
-
-  &:hover,
-  &:focus,
-  &:focus-within {
-    background-color: ${({ theme }) =>
-      theme.DEPRECATED_THEME.colors.foreground.gray16};
-    color: ${({ theme }) => theme.DEPRECATED_THEME.colors.black};
-  }
-`;
+export const NewStoryButton = styled(Button)(
+  ({ theme }) => css`
+    margin-bottom: 0;
+    margin-top: 0;
+    border: 1px solid ${theme.colors.border.defaultActive};
+    border-radius: ${theme.borders.radius.medium};
+
+    :hover {
+      color: ${theme.colors.bg.primary};
+      background-color: ${theme.colors.interactiveBg.brandPress};
+      border-color: ${theme.colors.interactiveFg.brandPress};
+      transition: border-color 0.25s linear, background-color 0.25s linear,
+        color 0.25s linear;
+    }
+  `
+);
 
 export const NavList = styled.ul`
   margin: 0;
   padding: 0;
 `;
 
 export const NavListItem = styled.li`
-  margin: 8px 0;
+  margin: 8px 28px;
   padding: 0;
   list-style-type: none;
 `;
 
-export const NavLink = styled.a`
-  ${TypographyPresets.Medium};
-  ${({ theme, active }) => `
-    display: block;
-    padding: 4px 28px;
-    margin: 4px 0;
-    font-weight: ${
-      theme.DEPRECATED_THEME.typography.weight[active ? 'bold' : 'normal']
-    };
-    text-decoration: none;
-    color: ${
-      active
-        ? theme.DEPRECATED_THEME.colors.gray900
-        : theme.DEPRECATED_THEME.colors.gray600
-    };
-
-    &:focus {
-      color: ${
-        active
-          ? theme.DEPRECATED_THEME.colors.gray900
-          : theme.DEPRECATED_THEME.colors.gray600
-      };
+export const NavButton = styled(Button)`
+  border-radius: 0;
+`;
+
+const activeLinkCss = css`
+  * {
```

Comment:
this is a lot of global selector usage ( `*` ) - is there any way to avoid it?

Reply:
yeah definitely, we can just target the children explicitly 👍